src/components/Map/map.js

```diff
@@ -93,7 +93,7 @@ class Map extends Component {
         id: layerName,
         source: "displayData",
         type: "symbol",
-        filter: ["==", property, layerName],
+        filter: ["==", property, 1],
```

Comment:
WDYT about inlining property and layerName in this method, since we always call it with the same arguments? So this method would take zero arguments.